packages/docs/src/theme/index.tsx

```diff
@@ -330,5 +330,21 @@ export default {
       },
     },
   },
+  accordion: {
+    accordionHeader: {
+      p: 2,
+      bg: 'transparent',
+      border: 'none',
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'space-between',
+      color: 'text',
+      width: '100%',
+      cursor: 'pointer',
+      '&:hover': {
+        bg: 'muted',
+      },
+    },
+  },
```

Comment:
Mislim da bi bilo dobro da se neki defaultni stilovi stave na accordion header komponentu sa sx propom...jer bez ikakvih stilova izgleda ko plain button?

Reply:
moze

Reply:
Sad kad si mi rekao za `__css` onda možda bolje umjesto `sx` staviti u `__css`?

Reply:
da, za core komponente je to ok. U svojim UserLand komponentama u projektu nebi trebao koristiti `__css` je ce potencijalno biti maknut u buducnosti... Pricao sam sa autorom :D